index.js

```diff
@@ -36,10 +36,17 @@ function isMatch(pattern, options) {
     };
   }
 
-  if (Array.isArray(pattern) || isObject(pattern)) {
+  if (Array.isArray(pattern)) {
+    return function (val) {
+      return mm(val, pattern, options).length !== 0;
```

Comment:
Mm, nope. I cant agree it would be the same as to now and adds totally different meaning, imo